tests/unitTests/material_model_tests/test_material_holzapfel_ogden_MA.cpp

```diff
@@ -0,0 +1,499 @@
+ * "Software"), to deal in the Software without restriction, including
+ * without limitation the rights to use, copy, modify, merge, publish,
+ * distribute, sublicense, and/or sell copies of the Software, and to
+ * permit persons to whom the Software is furnished to do so, subject
+ * to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS
+ * IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED
+ * TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A
+ * PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER
+ * OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+ * EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+ * PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+ * PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+ * LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+ * NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+#include "test_material_holzapfel_ogden_MA.h"
+
+// ----------------------------------------------------------------------------
+// ------------- Holzapfel-Ogden (Modified Anisotropy) Material  --------------
+// ----------------------------------------------------------------------------
+
+/**
+ * @brief Test fixture class for the Holzapfel-Ogden (Modified Anisotropy) material model.
+ * 
+ * This class sets up the necessary parameters and objects for testing the Holzapfel-Ogden (Modified Anisotropy) material model.
+*/
+class HolzapfelOgdenMATest : public MaterialTestFixture {
+protected:
+    // Material parameters object
+    HolzapfelOgdenMAParams params;
+
+    // Add the test object
+    TestHolzapfelOgdenMA* TestHO_ma;
+
+    // Setup method to initialize variables before each test
+    void SetUp() override {
+
+        MaterialTestFixture::SetUp();
+
+        // Set Holzapfel-Ogden parameters from cardiac benchmark paper
+        params.a = 59.0; // Pa
+        params.a_f = 18472.0; // Pa
+        params.a_s = 2481.0; // Pa
+        params.a_fs = 216.0; // Pa
+        params.b = 8.023; // no units
+        params.b_f = 16.026; // no units
+        params.b_s = 11.12; // no units
+        params.b_fs = 11.436; // no units
+        params.k = 100.0; // no units
+
+        // Set random values for f between 0 and 1 and normalize
+        params.f[0] = getRandomDouble(0.0, 1.0);
+        params.f[1] = getRandomDouble(0.0, 1.0);
+        params.f[2] = getRandomDouble(0.0, 1.0);
+        double norm_f = sqrt(params.f[0]*params.f[0] + params.f[1]*params.f[1] + params.f[2]*params.f[2]);
+        params.f[0] /= norm_f; params.f[1] /= norm_f; params.f[2] /= norm_f;
+
```

Comment:
Using `std::array` for `params.f` would allow more compact and cleaner code, get rid of all of the indexing
```
auto norm = sqrt(std::inner_product(f.begin(), f.end(), f.begin(), 0));
std::for_each(f.begin(), f.end(), [prod](double& n) { n /= prod; });
```